src/nsidc/metgen/metgen.py

```diff
@@ -361,13 +345,10 @@ def null_operation(configuration: config.Config, granule: Granule) -> Granule:
 
 def granule_collection(configuration: config.Config, granule: Granule) -> Granule:
     """
-    Find the Granule's Collection and add it to the Granule.
+    Associate the Collection with the Granule.
     """
-    # TODO: When we start querying CMR, refactor the pipeline to retrieve
-    # collection information from CMR once, then associate it with each
```

Comment:
My changes are intended to deal with this TODO. :-)

Reply:
Yay, good time to deal with this!